src/api/Gemfile

```diff
@@ -85,6 +85,13 @@ gem 'deep_cloneable', '~> 2.4.0'
 # Server-side datatables
 gem 'ajax-datatables-rails'
 
+# SSO
+gem 'omniauth'
+gem 'omniauth-gitlab'
+gem 'omniauth-github'
+gem 'omniauth_openid_connect', git: 'https://github.com/m0n9oose/omniauth_openid_connect', ref: '88ae46e968dec5e66d4f92773b52babbe72d41fe'
```

Comment:
What is the reason for pinning this gem to a commit? Is it a lack of a recent release?

Reply:
Yes, at the time when this was written, there was no compatible release.